desktop/ui/src/main/java/org/datacleaner/windows/OptionsDialog.java

```diff
@@ -19,33 +19,38 @@
  */
 package org.datacleaner.windows;
 
-import java.awt.*;
+import java.awt.BorderLayout;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.GridBagConstraints;
+import java.awt.Image;
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 import java.io.File;
-import java.io.IOException;
-import java.io.InputStream;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.text.NumberFormat;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
 import javax.inject.Inject;
-import javax.swing.*;
+import javax.swing.JButton;
+import javax.swing.JCheckBox;
+import javax.swing.JComponent;
+import javax.swing.JFileChooser;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JPasswordField;
+import javax.swing.JTextField;
+import javax.swing.Timer;
```

Comment:
I think change this is actually a place where IntelliJ and Eclipse formatters deviate.Eclipse default is 99, while IntelliJ has 5. I've changed mine to 30 at some point to avoid clashes.

However, I also think the the IntelliJ default is superior here; when you import many classes from a package, importing every single package seems a bit silly.

We will probably need to find a standard, changing it back and forth will annoy everyone. What it is isn't super important to me when it is automated anyway.